src/NServiceBus.Persistence.AzureStorage/Subscriptions/AzureStorageSubscriptionPersistence.cs

```diff
@@ -48,7 +49,7 @@ protected override void Setup(FeatureConfigurationContext context)
             }
 
             var subscriptionStorage = new AzureSubscriptionStorage(subscriptionTableName, connectionString, cacheFor);
-            context.Container.RegisterSingleton(typeof(ISubscriptionStorage), subscriptionStorage);
+            context.Services.AddSingleton(typeof(ISubscriptionStorage), subscriptionStorage);
```

Comment:
Isn't there an overload like the following?

```suggestion
            context.Services.AddSingleton<ISubscriptionStorage>(subscriptionStorage);
```

Reply:
yes, that's also possible. Pushed that change.